leapp/snactor/fixture.py

```diff
@@ -95,13 +95,16 @@ def feed(self, *models):
         for model in models:
             self._messaging.feed(model, self)
 
-    def run(self):
+    def run(self, config_model=None):
         """
         Execute the current actor.
 
```

Comment:
Once you're done, just don't forget to update the docstring here :-)

Reply:
@vinzenz  Sure thing. I changed the code a little in order to not force people to "feed()" the config. 

So in the test it would just be:

```
current_actor_context.run(mock_configs.CONFIG_ALL_SIGNED)
```

Could you please take a look at it whether it is not too hacky (probably is)?

Reply:
seems fine to me